snmp/datadog_checks/snmp/data/default_profiles/_generic-ups.yaml

```diff
@@ -0,0 +1,168 @@
+metric_tags:
+  - OID: 1.3.6.1.2.1.33.1.1.1
+    symbol: upsIdentManufacturer
+    tag: ups_ident_manufacturer
+  - OID: 1.3.6.1.2.1.33.1.1.2
+    symbol: upsIdentModel
+    tag: ups_ident_model
+  - OID: 1.3.6.1.2.1.33.1.1.5
+    symbol: upsIdentName
+    tag: ups_ident_name
+
+metrics:
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.1.0
+      name: upsBatteryStatus
+      mapping:
+        1: unknown
+        2: batteryNormal
+        3: batteryLow
+        4: batteryDepleted
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.2.0
+      name: upsSecondsOnBattery
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.3.0
+      name: upsEstimatedMinutesRemaining
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.4.0
+      name: upsEstimatedChargeRemaining
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.5.0
+      name: upsBatteryVoltage
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.6.0
+      name: upsBatteryCurrent
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.2.7.0
+      name: upsBatteryTemperature
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.3.1.0
+      name: upsInputLineBads
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.3.2.0
+      name: upsInputNumLines
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.4.1.0
+      name: upsOutputSource
+      mapping:
+        1: other
+        2: none
+        3: normal
+        4: bypass
+        5: battery
+        6: booster
+        7: reducer
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.4.2.0
+      name: upsOutputFrequency
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.4.3.0
+      name: upsOutputNumLines
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.5.1.0
+      name: upsBypassFrequency
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.5.2.0
+      name: upsBypassNumLines
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.6.1.0
+      name: upsAlarmsPresent
+  - MIB: UPS-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.33.1.7.3.0
+      name: upsTestResultsSummary
+      mapping:
+        1: donePass
+        2: doneWarning
+        3: doneError
+        4: aborted
+        5: inProgress
+        6: noTestsInitiated
+  # - MIB: UPS-MIB
+  #   symbol:
+  #     OID: 1.3.6.1.2.1.33.1.7.4.0
+  #     name: upsTestResultsDetail
```

Comment:
Could you add a comment why it's currently ignored/skipped?

Maybe:

> enum in scalar metric is not supported yet (keep this metric and this comment in profile until it's fixed)

Reply:
Added :)